README.md

```diff
@@ -15,7 +15,7 @@ You can view the most-upvoted feature requests with [this link](https://github.c
 
 ## Getting help
 
-Please open an issue or stop be #ec-addon-docs in Slack!
+Please open an issue or stop by #ec-addon-docs in Slack!
```

Comment:
Since you are adding this file, we might want to update the reference to slack as the Ember community has moved to Discord.  https://github.com/emberjs/rfcs/pull/345

Reply:
Good call!  Discord link is  https://discord.gg/zT3asNS.

Maybe change this line to 'Please open an issue or join the [Ember community on Discord]( https://discord.gg/zT3asNS )'